packages/grafana-prometheus/rollup.config.ts

```diff
@@ -0,0 +1,54 @@
+import image from '@rollup/plugin-image';
+import resolve from '@rollup/plugin-node-resolve';
+import path from 'path';
+import copy from 'rollup-plugin-copy';
+import dts from 'rollup-plugin-dts';
+import esbuild from 'rollup-plugin-esbuild';
+import { externals } from 'rollup-plugin-node-externals';
+
+const icons = require('../../public/app/core/icons/cached.json');
```

Comment:
I'm interested in understanding why there is a need to bundle all the icons in this package? Icon usage comes from the `<Icon />` component in `@grafana/ui` which in turn already uses the contents of this cache via Grafana core.

Reply:
I can remove the icons, those were left over from the PromQAIL work and needed to be moved. We don't have to have them in the package. I'll get a PR for that up today.